### ISSUE TITLE
Cannot Import CRD with address selector

### ISSUE THREAD

**Issue by corang** [neuvector/neuvector#102]
**Environment**
Platform: Kubernetes EKS
Kubernetes/Platform Version(s): v1.21.5-eks-bc4871b
NeuVector version: v5.0.0-preview.2

**Describe the bug**
Importing a CRD that includes an address selector under an NVClusterSecurityRule always fails with `Import failed : Group Policy Rule format error: Group Google-group with address criterion cannot have WAF policy`. This is true with a directly exported yaml file and with a modified one where the `waf` key is deleted.

**To Reproduce**
1. Create a new group with an `address=google.com` criteria
2. Export that group
3. Import the newly exported file
4. Error
5. Remove `waf` key as directed
6. Import modified file
7. Error

**Expected behavior**
A freshly exported file should be importable without modification or at least have an error that applies to the problem in the file.

**Comment by corang** [comment]
Example exported CRD
```
apiVersion: v1
items:
- apiVersion: neuvector.com/v1
  kind: NvClusterSecurityRule
  metadata:
    name: 0oogle-group
    namespace: ""
  spec:
    egress: []
    file: []
    ingress:
    - action: deny
      applications:
      - any
      name: Google-group-ingress-0
      ports: any
      priority: 0
      selector:
        comment: ""
        criteria:
        - key: service
          op: =
          value: busybox.struts
        - key: domain
          op: =
          value: struts
        name: nv.busybox.struts
        original_name: ""
    process: []
    process_profile:
      baseline: default
    target:
      policymode: N/A
      selector:
        comment: ""
        criteria:
        - key: address
          op: =
          value: google.com
        name: Google-group
        original_name: ""
    waf: null
kind: List
metadata: null
```

**Comment by chunyenlin** [comment]
Thanks for the bug reporting.
By design IP groups do not have WAF settings.
However, there is a bug in v5.0.0-preview.2 that IP groups without WAF settings cannot be imported thru CRD.
It will be fixed in 5.0.0
